### PR TITLE
Publish artifacts in the PR when everything is green 

### PR DESCRIPTION
**Pull request by adi88d**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/27000

## Description
Add the build summary for successful packs to the pull request when there is no failed packs,  inside `Upload Packs To Marketplace Storage` step.

## Screenshots
image

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Tests/Marketplace/upload_packs.py [hunk 1]
```diff
@@ -628,6 +669,49 @@ def option_handler():
...
+    Args:
+        comment (string): The comment text.
+
+    """
+    token = os.environ['CONTENT_GITHUB_TOKEN']
+    branch_name = os.environ['CIRCLE_BRANCH']
+    sha1 = os.environ['CIRCLE_SHA1']
+
+    print('branch_name = ' + branch_name)
+    print('sha1 = ' + sha1)
```
Comment (yaakovi) [inline Tests/Marketplace/upload_packs.py:684]: remove all debug prints

File: Tests/Marketplace/upload_packs.py [hunk 2]
```diff
@@ -628,6 +669,49 @@ def option_handler():
...
+
+    query = '?q={}+repo:demisto/content+org:demisto+is:pr+is:open+head:{}+is:open'.format(sha1, branch_name)
```
Comment (yaakovi) [inline Tests/Marketplace/upload_packs.py:686]: use f-strings
if you stated `repo`, is `org` required?
```suggestion
    query = f'?q={sha1}+repo:demisto/content+org:demisto+is:pr+is:open+head:{branch_name}'
```

**Comment by Itay4** [comment]
this need to be only for external PRs

File: Tests/Marketplace/upload_packs.py (hunk 2 above)
Comment (adi88d) [reply Tests/Marketplace/upload_packs.py:686]: org is not required, i removed it

**Review by yaakovi — APPROVED** [review]

**Pull request merged**